client/main.cpp

```diff
@@ -64,6 +64,8 @@
 
 using namespace std;
 
+PortCustomConfig ahah;
```

Comment:
ahah is a bad name for a variable.

Also I'm not comfortable making this a global.